src/Symfony/Component/Console/Command/Command.php

```diff
@@ -495,13 +495,10 @@ public function getName()
 
     /**
      * @param bool $hidden Whether or not the command should be hidden from the list of commands
-     *                     The default value will be true in Symfony 6.0
      *
      * @return Command The current instance
-     *
-     * @final since Symfony 5.1
      */
-    public function setHidden(bool $hidden /*= true*/)
+    final public function setHidden(bool $hidden = true)
```

Comment:
Since it's final now… Shall we?

```suggestion
    final public function setHidden(bool $hidden = true): static
```